<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebcacheOptionsPage</class>
 <widget class="QWidget" name="WebcacheOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>240</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="webcache_enabled">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="title">
      <string>Web Cache</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <item row="6" column="1">
       <layout class="QHBoxLayout" name="hlayout_max_size">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="max_size_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Maximum disk cache requested:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="webcache_max_size">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="suffix">
           <string>MB</string>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>250</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="hSpacer_max_size">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="11" column="1">
       <spacer name="vSpacer_internal">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="1" rowspan="2">
       <layout class="QHBoxLayout" name="hLayout_clear_cache">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="webcache_current_sizes">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">You are currently using xMB of a maximum of yMB.</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="webcache_clear_cache">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Clear Cache</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="hSpacer_clear_cache">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="12" column="1">
       <layout class="QHBoxLayout" name="hLayout_force_cache">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="force_cache_label">
          <property name="text">
           <string>Force cache usage:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>3</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="webcache_force_cache">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="hSpacer_force_cache">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="13" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Note: If you force cache usage, data will always be read from the cache if it is available. To get updated data from the web you will need to clear this setting or clear the cache.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="note">
     <property name="text">
      <string>Note: Please read the help file and understand what they do before enabling these settings.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="vSpacer_bottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>webcache_enabled</tabstop>
  <tabstop>webcache_max_size</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
